### PR TITLE
service mesh pattern maintainers added 

### PR DESCRIPTION
**Pull request by legendarykamal**
Signed-off-by: Kamal Nayan <geniusamansingh@gmail.com>

**Description**

Service mesh pattern maintainers added 

Screenshot from 2022-08-29 07-44-50
**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1bf6c0a595bf51e8a618453d108d6dfb7c6bca2d at: https://630c25ef6d86c23f93968b1a--layer5.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -99,6 +99,15 @@ export const repo_data = [
         link: ["https://layer5.io/community/members/ekene-leonard-nwobodo"],
         repository: " https://discuss.layer5.io",
       },
+      {
+        project: "Service Mesh Pattern",
+        image: patterns,
+        site: "https://github.com/service-mesh-patterns/service-mesh-patterns",
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/repo-data.js:105]: Let's use the actual site URL. No harm in that

Comment (legendarykamal) [reply src/sections/Community/Handbook/repo-data.js:105]: This one -> https://meshery.io/ ?

Comment (Nikhil-Ladha) [reply src/sections/Community/Handbook/repo-data.js:105]: Nope.
https://service-mesh-patterns.github.io/service-mesh-patterns/

Comment (legendarykamal) [reply src/sections/Community/Handbook/repo-data.js]: oh github pages , that was silly on my part :sweat_smile:

File: src/sections/Community/Handbook/repo-data.js [hunk 2]
```diff
@@ -99,6 +99,15 @@ export const repo_data = [
         link: ["https://layer5.io/community/members/ekene-leonard-nwobodo"],
         repository: " https://discuss.layer5.io",
       },
+      {
+        project: "Service Mesh Pattern",
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/repo-data.js:103]:
```suggestion
        project: "Service Mesh Patterns",
```

**Pull request merged**